### ISSUE TITLE
Do an accessibility review of our processes and tools, and implement improvements.

### ISSUE THREAD

**Issue by stefpiatek** [socrse/edia#19]
We should do this if and update the [EDIA strategy](https://docs.google.com/document/d/1YCrPOEVjHjd_I5Hq9GYNXyzGmlzxAeBFC-4XN03g_Jg) document.

See sub issues for quotes for website and tools and processes

**Comment by stefpiatek** [comment]
Emailed [understood](https://www.understood.org/en/workplace), [shaw trust](https://shawtrust.org.uk/accessibility-services/#contact) and [diversity trust](https://www.diversitytrust.org.uk/accessibility-audit/) to find out more about what they could offer and costs

**Comment by Robadob** [comment]
> Emailed [workplace](https://www.understood.org/en/workplace), [shaw trust](https://shawtrust.org.uk/accessibility-services/#contact) and [diversity trust](https://www.diversitytrust.org.uk/accessibility-audit/) to find out more about what they could offer and costs

~~workplace~~ understood

**Comment by stefpiatek** [comment]
Have started chatting with Shaw trust and diversity trust. understood aren't taking on any more accessibility reviews. Should think about whether we want to prioritise the website or RSESoc charity processess and procedures. Maybe we just do both

- website and WCAG AA certification
  - Much more automatible, but still takes work
  - Would still need action the changes
  - Is the most externally facing part, along with slack
- Processess and procedures
  - Would benefit from external review and recommendations
  - We want to increase diveristy of trustees and inclusitivity there
- Conference guidance, procedures and policies

other work in this space that we discussed:

- https://www.ukri.org/publications/epsrc-guidance-for-neuro-inclusive-meetings-and-events/epsrc-neuro-inclusion-audit-recommendations-and-guidance/
